recipes/mono/mono-osx.recipe

```diff
@@ -6,7 +6,7 @@ from cerbero.tools.osxrelocator import OSXRelocator
 
 class Recipe(recipe.Recipe):
     name = 'mono'
-    version = '5.2.0'
+    version = '5.12.0'
```

Comment:
do we need another folder path to make the old versions remain after updating the mono framework?